cmd/registry/cmd/compute-lint.go

```diff
@@ -89,17 +96,28 @@ func (task *computeLintTask) Run() error {
 	if err != nil {
 		return err
 	}
-	relation := "lint"
-	log.Printf("computing %s/properties/%s", spec.Name, relation)
+	var relation string
 	var lint *rpc.Lint
 	if strings.HasPrefix(spec.GetStyle(), "openapi") {
-		lint, err = core.NewLintFromOpenAPI(spec.Name, spec.GetContents())
+		// the default openapi linter is gnostic
+		if task.linter == "" {
+			task.linter = "gnostic"
+		}
+		relation = "lint-" + task.linter
```

Comment:
Yes, I think this is generally a good idea. A type seemed like a lot to add for this, but I added helper functions to build the relation names so that the name generation is in one place.